app/controllers/console_controller.rb

```diff
@@ -0,0 +1,11 @@
+# This controller is used by the console_updater js to retrieve logs
+# for a specific job
+class ConsoleController < ProjectScopedController
```

Comment:
Can we card using Console#status for Upload and Export (report and validate) too?

Also, maybe `JobsController` would be a better name as it can allow us to grow in the future to a Jobs#index, etc.

Food for thought.

Reply:
@etdsoft  I wanted to use `jobs`, but we have a collision with `resque` then?